models/blue_sea/src/dataloaders/fetch_data_run_query.py

```diff
@@ -0,0 +1,9 @@
+from ..dataloaders.queryset import get_querysets
+from pathlib import Path
+
+
+def fetch_data():
+    data = get_querysets()
+    data.to_parquet(
+        f"{Path(__file__).parent.parent.parent}/data/processed/processed.parquet")
```

Comment:
As mentioned in baseline_pgm PR, this will go into config in future.. but fine for now